pkg/reconciler/pipelinerun/pipelinerun_test.go

```diff
@@ -1290,8 +1290,8 @@ status:
           image: busybox
           script: 'exit 0'
   conditions:
-  - message: "Tasks Completed: 2 (Failed: 1, Cancelled 0), Skipped: 0"
-    reason: Failed
+  - message: "Tasks Completed: 1 (Failed: 1, Cancelled 0), Skipped: 0"
+    reason: PipelineValidationFailed
```

Comment:
NIT: I find this slightly confusing. The `PipelineRun`, made of two tasks, is finished, with 1 `Task` completed (failed) and nothing else. The reason "PipelineValidationFailed" helps a bit to explain what happens, but not fully. 
Perhaps we could consider the second `TaskRun` as failed or add a new group "Validation Failed" e.g.

- `Tasks Completed: 1 (Failed: 1, Cancelled 0), Skipped: 0, Failed Validation: 1`

If we wanted to avoid adding another group, we could count the failed validation as failed:

- `Tasks Completed: 2 (Failed: 2, Cancelled 0), Skipped: 0`


@divyansh42 @vdemeester wdyt?

Reply:
Ah makes sense ! I do like the idea of having the `Failed Validation` group.

Reply:
Thanks for bringing this up @afrittoli. I also had this thought earlier. 
So from the example that @afrittoli is referencing the complete count is 1 and the Failed task is the one that failed in the validation.
I included validationFailed in the failed task as I see in the code `IgnoredFailed` is also included in the `Failed` count so followed the same approach.
I do like the idea of adding the FailedValidation count.

Reply:
@afrittoli I have made the required changes. Could you please take a look?